source/fundamentals/write-data.txt

```diff
@@ -0,0 +1,146 @@
+.. _entity-framework-write-data:
+
+=====================
+Write Data to MongoDB
+=====================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: EF, EF Core, code example, write
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+{+framework-core+} allows you to work with data in your application without
+explicitly running database commands. You can perform write operations to
+insert, update, or delete data within your application, and then persist those
+changes to MongoDB by using the ``SaveChanges()`` method.
+
+When you call the ``SaveChanges()`` method, the {+provider-short+} automatically detects
+any changes made to your data and runs the necessary commands to update the
+database by using the MongoDB Query API.
+
+In this guide, you can see examples of how to perform common write operations on
+an application configured to use the {+provider-long+}.
+
+.. tip::
+
+   To learn how to configure an application to use the {+provider-short+}, see
+   :ref:`entity-framework-configure`.
+
+Insert
+------
+
+You can use the ``Add()`` method to insert a single entity into your collection, 
+or you can use the ``AddRange()`` method to insert multiple entities at once. 
+
+Insert One Entity
+~~~~~~~~~~~~~~~~~
+
+The ``Add()`` method accepts a single entity of the same type that you
+specified on the ``DbSet`` instance that you are modifying.
+
+The following code uses the ``Add()`` method to add a new ``Planet`` object to
+the ``DbSet`` called ``Planets``. It then calls the ``SaveChanges()`` method to
+insert that entity into the MongoDB collection.
+
+.. literalinclude:: /includes/code-examples/quick-reference.cs
+   :start-after: // start-insert-one
+   :end-before: // end-insert-one
+   :language: csharp
+   :dedent:
+
+Insert Multiple Entities
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``AddRange()`` method accepts an array of entities that you want to add to the ``DbSet``. 
+
+The following code uses the ``AddRange()`` method to add an array of ``Planet``
+objects to the ``DbSet`` called ``Planets``. It then calls the ``SaveChanges()``
+method to insert those entities into the MongoDB collection.
+
+.. literalinclude:: /includes/code-examples/quick-reference.cs
+   :start-after: // start-insert-many
+   :end-before: // end-insert-many
+   :language: csharp
+   :dedent:
+
+Update
+------
+
+To update an entity, first retrieve the entity that you want to update. Then
+make the changes to that entity. To save the update to MongoDB, call the
```

Comment:
I'd add "using the properties to new values or by adding and removing items from the lists or arrays the entity has."

Reply:
I tried to keep the "make the changes" part pretty vague so it didn't seem like we are saying that there are only certain specific types of updates supported with the provider. I think the main focus of this section should be that they have to call `SaveChanges()` after any updates they do, since that's what's specific to EF. 
Happy to workshop a little more here or through slack if you still think we should get more specific on that part!

Reply:
I think we could clarify that anything changing the entity - setting properties or adding/removing items from any collections on it - will be reflected via SaveChanges thanks to the object tracking.

Reply:
I added a clarifying sentence, let me know if that covers what you have in mind.